api/envoy/config/trace/v2/trace.proto

```diff
@@ -130,9 +130,6 @@ message OpenCensusConfig {
   // The URL to Zipkin, e.g. "http://127.0.0.1:9411/api/v2/spans"
   string zipkin_url = 6;
 
-  // The Zipkin service name.
-  string zipkin_service_name = 7;
```

Comment:
Can you reserve this? I guess we don't need to bother with deprecation since this just shipped?

Reply:
Done.